### ISSUE TITLE
Option to block URL opens from games

### ISSUE THREAD

**Issue by colin969** [ruffle-rs/ruffle#838]
A lot of games trigger popups / browser opens for their websites. This is unusual behavior by modern standards and sort of intrusive.

SWF example:
https://unstable.life/Flashpoint/Server/htdocs/localflash/1760__commando.swf

Run in the standalone application, click on the screen during the startup and it'll open up a link to miniclip.com in your default browser.

It'd be nice to have a way to block this behavior given it never added anything to 99% of games in the first place.

**Comment by desertking** [comment]
Yes i thought the same, there should be an option that ruffle detects allowNetworking="internal" and blocks external requests etc as the original did

**Comment by FoxhackDN** [comment]
I would like to mention something I just noticed while testing the Strong Bad emails at archive.org - when you watch one and it ends, sometimes it has a "Back" option, so I clicked it thinking it would go back to the beginning. Instead it tried to take me to another page - a hardcoded relative link to 'sbemail.html'.

It would be a good idea to add an option to block those kinds of clickable links too; or at the very least offer a way to disable clickthroughs depending on the configuration set by the uploader.
